lib/fpm/command.rb

```diff
@@ -117,6 +117,9 @@ def help(*args)
     "configuration files, specify this flag multiple times.  If argument is " \
     "directory all files inside it will be recursively marked as config files.",
     :multivalued => true, :attribute_name => :config_files
+  option "--no-config-files", :flag,
```

Comment:
Given this is a debian-specific patch, I'm have hesitations on having it listed as a default flag and not deb-specific. Thoughts?